enable/api.py

```diff
@@ -1,6 +1,161 @@
+===========
+
+- :class:`~.IDroppedOnHandler`
+- :func:`~.str_to_font`
+- :func:`~.intersect_bounds`
+
+Constants
+---------
+- :attr:`~.TOP`
+- :attr:`~.VCENTER`
+- :attr:`~.BOTTOM`
+- :attr:`~.LEFT`
+- :attr:`~.HCENTER`
+- :attr:`~.RIGHT`
+- :attr:`~.TOP_LEFT`
+- :attr:`~.TOP_RIGHT`
+- :attr:`~.BOTTOM_LEFT`
+- :attr:`~.BOTTOM_RIGHT`
+- :attr:`~.empty_rectangle`
+
+Enable Trait Types
+==================
+
+- :attr:`~.border_size_editor`
+- :attr:`~.font_trait`
+- :attr:`~.bounds_trait`
+- :attr:`~.ComponentMinSize`
+- :attr:`~.ComponentMaxSize`
+- :attr:`~.Pointer`
+- :attr:`~.cursor_style_trait`
+- :attr:`~.spacing_trait`
+- :attr:`~.padding_trait`
+- :attr:`~.margin_trait`
+- :attr:`~.border_size_trait`
+- :attr:`~.TimeInterval`
+- :attr:`~.Stretch`
+- :attr:`~.NoStretch`
+- :attr:`~.LineStyle`
+- :attr:`~.LineStyleEditor`
+
+Constants
+---------
+- :attr:`~.basic_sequence_types`
+- :attr:`~.sequence_types`
+- :attr:`~.pointer_shapes`
+- :attr:`~.CURSOR_X`
+- :attr:`~.CURSOR_Y`
+- :attr:`~.cursor_styles`
+
+Colors
+======
+
+- :attr:`~.color_table`
+- :attr:`~.transparent_color`
+- :class:`~.ColorEditorFactory`
+
+Color Trait Types
+-----------------
+
+- :class:`~.ColorTrait`
+- :attr:`~.black_color_trait`
+- :attr:`~.white_color_trait`
+- :attr:`~.transparent_color_trait`
+
+Markers
+=======
+
+- :class:`~.SquareMarker`
+- :class:`~.CircleMarker`
+- :class:`~.TriangleMarker`
+- :class:`~.Inverted_TriangleMarker`
+- :class:`~.LeftTriangleMarker`
+- :class:`~.RightTriangleMarker`
+- :class:`~.PentagonMarker`
+- :class:`~.Hexagon1Marker`
+- :class:`~.Hexagon2Marker`
+- :class:`~.StarMarker`
+- :class:`~.CrossPlusMarker`
+- :class:`~.PlusMarker`
+- :class:`~.CrossMarker`
+- :class:`~.DiamondMarker`
+- :class:`~.DotMarker`
+- :class:`~.PixelMarker`
+- :class:`~.CustomMarker`
+- :class:`~.AbstractMarker`
+
+Marker Trait Types
+------------------
+- :class:`~.MarkerTrait`
+- :attr:`~.marker_trait`
+
+Marker Constants
+----------------
+- :attr:`~.MarkerNameDict`
+- :attr:`~.marker_names`
+
+Events
+======
+
+- :class:`~.BasicEvent`
+- :class:`~.BlobEvent`
+- :class:`~.BlobFrameEvent`
+- :class:`~.DragEvent`
+- :class:`~.KeyEvent`
+- :class:`~.MouseEvent`
+
+Event Trait Types
+-----------------
+
+- :attr:`~.drag_event_trait`
+- :attr:`~.key_event_trait`
+- :attr:`~.mouse_event_trait`
+
+Enable Components
+=================
+
+- :class:`~.Interactor`
+- :class:`~.BaseTool`
+- :class:`~.KeySpec`
+- :class:`~.AbstractOverlay`
+- :class:`~.Canvas`
+- :class:`~.Component`
+- :class:`~.Container`
+- :class:`~.CoordinateBox`
+- :class:`~.ComponentEditor`
+- :class:`~.OverlayContainer`
+- :class:`~.ConstraintsContainer`
+- :class:`~.Label`
+- :class:`~.GraphicsContextEnable`
+- :class:`~.ImageGraphicsContextEnable`
+
+Enable Widgets
+==============
+
+- :class:`~.AbstractWindow`
+- :class:`~.NativeScrollBar`
+- :class:`~.Compass`
+- :class:`~.Scrolled`
+- :class:`~.Slider`
+- :class:`~.TextFieldStyle`
+- :class:`~.TextField`
+- :class:`~.TextFieldGrid`
```

Comment:
Most of these are neglected and of limited usefulness. I would recommend their omission here and schedule them for deprecation.